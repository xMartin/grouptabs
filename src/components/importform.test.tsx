import renderer from "react-test-renderer";
import ImportForm from "./importform";

it("renders form", () => {
  const tree = renderer
    .create(
      <ImportForm tabId="xy123hq" onTabIdChange={vi.fn()} onSubmit={vi.fn()} />,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
