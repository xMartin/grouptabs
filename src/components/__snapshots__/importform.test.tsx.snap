// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`renders form 1`] = `
<form
  className="import-form"
  onSubmit={[Function]}
>
  <div
    className="row-label"
  >
    Import group:
  </div>
  <input
    autoFocus={true}
    className="full-width"
    onChange={[Function]}
    placeholder="Group ID â€¦"
    type="text"
    value="xy123hq"
  />
  <button>
    Import
  </button>
  <div
    className="error-message"
  />
</form>
`;
